custom_components/tuya_local/devices/nine_ev_charger.yaml

```diff
@@ -0,0 +1,262 @@
+name: EV charger
+products:
+  - id: bktb3jskdic1ar2t
+    name: Nine 32A EV charger
+primary_entity:
+  entity: sensor
+  name: Charger State
+  icon: "mdi:ev-station"
+  class: enum
+  dps:
+    - id: 3  # work_state
+      type: string
+      name: sensor
+      mapping:
+        - dps_val: charger_charging
+          value: Charging
+        - dps_val: charger_free
+          value: Available
+        - dps_val: charger_insert
+          value: Plugged in
+        - dps_val: charger_free_fault
+          value: Fault
+        - dps_val: charger_wait
+          value: Plugged in, not charging
+        - dps_val: charger_pause
+          value: Paused
+        - dps_val: charger_end
+          value: Charge finished
+        - dps_val: charger_fault
+          value: Fault (plugged in)
+    - id: 23  # system_version
+      type: string
+      name: system_version
+secondary_entities:
+  - entity: number
+    name: Charging current
+    category: config
+    class: current
+    icon: "mdi:ev-plug-type2"
+    dps:
+      - id: 4  # charge_cur_set
+        type: integer
+        name: value
+        unit: A
+        optional: true
+        range:
+          min: 6
+          max: 80
+        mapping:
+          - scale: 1
```

Comment:
Is this a tuya `scale: 1`? Is the range 6-80A, or 0.6-8.0A?

Reply:
Scale 1, from 6 to 80.

Reply:
In that case, the mapping can be removed, 1 is the default scale.